tests/test_kafka.py

```diff
@@ -356,7 +356,7 @@ def test_jmx_bridged_network(self):
             image="confluentinc/cp-jmxterm",
             command=JMX_CHECK.format(jmx_hostname="kafka-bridged-jmx", jmx_port="9999"),
             host_config={'NetworkMode': 'standalone-network-test_zk'})
-        self.assertTrue("Version = 0.11.0.0-cp1;" in logs)
+        self.assertTrue("Version = 0.11.0.1-cp4;" in logs)
```

Comment:
Hmm, why is it cp4?

Reply:
That is the version in the actual docker image (for RC4?). I had to change it for running the tests. Since this is the 3.3.x branch, I thought it made sense to update the test to use that version. I don't know if the image is regenerated with different versions later or whether this is the Kafka version that the image will be shipped with.

Reply:
@aayars do you know?

Reply:
I think cp4 was used erroneously for rc4, but when we released, it was corrected to cp1.

Reply:
@rajinisivaram, maybe you can revert this change then?